### PR TITLE
update 0.10.0

### PR DESCRIPTION
**Pull request by agourlay**
Contains:
- batch search & recommendation
- add builder for points with named vector

### CODE REVIEW

File: src/client.rs [hunk 1]
```diff
@@ -343,23 +377,34 @@ impl QdrantClient {
...
-            with_vector,
-            with_payload: with_payload.map(|x| x.into()),
-        }).await?;
+        let result = points_api
+            .get(GetPoints {
+                collection_name: collection_name.to_string(),
+                ids: points,
+                with_vector,
+                with_payload: with_payload.map(|x| x.into()),
+                with_vectors: None,
```
Comment (agourlay) [inline src/client.rs:386]: I think I need to enable users to use the named vector selector here.

**Comment by agourlay** [comment]
integration tests need to run against the 0.10.0 Docker container

File: tests/integration-tests.sh [hunk 2]
```diff
@@ -11,7 +11,7 @@ function stop_docker()
 # Ensure current path is project root
 cd "$(dirname "$0")/../"
 
-QDRANT_VERSION='v0.9.0'
+QDRANT_VERSION='v0.9.1'
```
Comment (IvanPleshkov) [inline tests/integration-tests.sh:14]: Do we need update version here to `0.10.0` and merge this PR after release?

Comment (agourlay) [reply tests/integration-tests.sh:14]: I was planning on updating this PR once the new Docker image is out.

Comment (IvanPleshkov) [reply tests/integration-tests.sh:14]: Okay, thanks for response!

**Pull request merged**